cmd/deploy/deploy.go

```diff
@@ -601,6 +627,14 @@ func (dc *DeployCommand) deployStack(ctx context.Context, opts *Options) error {
 	defer oktetoLog.SetStage("")
 	composeSectionInfo := opts.Manifest.Deploy.ComposeSection
 	composeSectionInfo.Stack.Namespace = okteto.Context().Namespace
+
+	// When endpoints are defined in both compose and okteto manifest, okteto manifest ones are deployed instead
+	if len(opts.Manifest.Deploy.Endpoints) > 0 && len(composeSectionInfo.Stack.Endpoints) > 0 {
+		oktetoLog.Warning("Endpoints are defined in the stack and in the manifest. The endpoints defined in the manifest will be used.")
+		oktetoLog.Warning("Endpoints defined at the stack manifest will be deprecated")
+		composeSectionInfo.Stack.Endpoints = nil
```

Comment:
I think we should have another structure to handle the flow of this data. I know we are doing the same in other places, but the relationship between `opts.Manifest.Deploy.ComposeSection.Stack.Endpoints` and `opts.Manifest.Deploy.Endpoints` is not trivial. As it is right now, we are delegating the responsibility to `opts.Manifest.Deploy.Endpoints`.

I think that we should take those 2 variables as different feeds for another structure like `DeployOptions.Endpoints`

Reply:
we are delegating into the manifest endpoints, as we are moving towards having the feature at this manifest, instead of having a compose with the extra feature (compose does not have this on the spec, is an okteto added feature for the stack)

Reply:
Maybe we can add do it in the function `inferFromStack()`

Reply:
> Maybe we can add do it in the function `inferFromStack()`

i did put it there... but by the time it got to deploy the stack... they where there again :o i will check